docs/Factory.md

```diff
@@ -153,3 +153,35 @@ Please note this setting this option to `false` goes against
 The header name used to know the request id. See [the request id](https://github.com/fastify/fastify/blob/master/docs/Logging.md#logging-request-id) section.
 
 + Default: `'request-id'`
+
+<a name="factory-trust-proxy"></a>
+### `trustProxy`
+
+By enabling the `trustProxy` option, faistify will have knowledge that it's sitting behind a proxy and that the X-Forwarded-* header fields may be trusted, which otherwise may be easily spoofed.
+
+```js
+const fastify = Fastify({ trustProxy: true })
+```
+
++ Default: `false`
++ `true/false`: Trust or not to trust all proxies.
++ `string`: Trust only given ip/net (e.g. `'127.0.0.1'`). Could be comma-separated values (e.g. `'127.0.0.1,192.168.1.1/24'`).
```

Comment:
"ip/net" => "IP/CIDR"
"Could be comma-separated values" => "May be a list of comma separated values"